pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +214,23 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("bindings", "endpoints", "persistentvolumes", "persistentvolumes/status",
```

Comment:
I would not expect global read/write access to these things... I would expect this role to grant the pv/pvc/storageclass permissions, then the user be granted an admin role in a particular namespace if they needed to set up secrets or endpoints for their provisioner.